src/test/java/teammates/ui/webapi/GetActionClassesActionTest.java

```diff
@@ -0,0 +1,146 @@
+                "SearchStudentsAction",
+                "FeedbackSessionRemindEmailWorkerAction",
+                "DeleteStudentProfilePictureAction",
+                "PostStudentProfilePictureAction",
+                "GetStudentProfilePictureAction",
+                "DeleteFeedbackSessionAction",
+                "CreateFeedbackSessionAction",
+                "GetFeedbackSessionAction",
+                "UpdateFeedbackSessionAction",
+                "FeedbackSessionClosingRemindersAction",
+                "GetLocalDateTimeInfoAction",
+                "GetTimeZonesAction",
+                "FeedbackSessionRemindParticularUsersEmailWorkerAction",
+                "GetFeedbackResponsesAction",
+                "SubmitFeedbackResponsesAction",
+                "FeedbackSessionPublishedEmailWorkerAction",
+                "FeedbackSessionClosedRemindersAction",
+                "SendErrorReportAction",
+                "GetActionClassesAction",
+                "UnpublishFeedbackSessionAction",
+                "PublishFeedbackSessionAction",
+                "GetSessionResultsAction",
+                "GetHasResponsesAction",
+                "DatastoreBackupAction",
+                "RestoreCourseAction",
+                "BinCourseAction",
+                "DeleteAccountAction",
+                "CreateAccountAction",
+                "GetAccountAction",
+                "FeedbackSessionPublishedRemindersAction",
+                "QueryLogsAction",
+                "SessionLinksRecoveryAction",
+                "SendJoinReminderEmailAction",
+                "RegenerateStudentCourseLinksAction",
+                "CompileLogsAction",
+                "GetAuthInfoAction",
+                "GetFeedbackSessionSubmittedGiverSetAction",
+                "GetCourseJoinStatusAction",
+                "JoinCourseAction",
+                "GetSessionResponseStatsAction",
+                "DeleteCourseAction",
+                "CreateCourseAction",
+                "GetCourseAction",
+                "UpdateCourseAction",
+                "GetFeedbackQuestionRecipientsAction",
+                "DowngradeAccountAction",
+                "RemindFeedbackSessionSubmissionAction",
+                "FeedbackSessionUnpublishedEmailWorkerAction",
+                "SendEmailWorkerAction",
+                "GetInstructorsAction",
+                "PutDataBundleDocumentsAction",
+                "FeedbackSessionResendPublishedEmailWorkerAction",
+                "StudentCourseJoinEmailWorkerAction",
+                "SearchInstructorsAction",
+                "GetCourseSectionNamesAction",
+                "ResetAccountAction",
+                "FeedbackSessionOpeningRemindersAction",
+                "DeleteStudentsAction",
+                "GetStudentsAction",
+                "EnrollStudentsAction",
+                "DeleteFeedbackQuestionAction",
+                "CreateFeedbackQuestionAction",
+                "UpdateFeedbackQuestionAction"
```

Comment:
This is still not right. This merely checks that the list of actions matches some other list, but nothing tells that the string corresponds to a valid action. What if `UpdateFeedbackQuestionAction` is not a valid action class name, but is included into the list by mistake?

Reply:
Understood. I have updated to test it in another way.